pkgs/by-name/or/oreo-cursors-plus/package.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  ruby,
+  inkscape,
+  xorg,
+  cursorsConf ? null, # If set to a string, overwrites contents of './cursors.conf'
+}:
+with lib;
+  stdenv.mkDerivation rec {
```

This `rec` doesn't seem to be used

```suggestion
  stdenv.mkDerivation {
```